torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1090,7 +1090,9 @@ def sample_inputs_cdist(op_info, device, dtype, requires_grad, **kwargs):
 
     samples = []
     for cm in ['use_mm_for_euclid_dist', 'donot_use_mm_for_euclid_dist']:
-        for p in [0, 1, 2, 3, 0.5, 1.5, 2.5, float("inf")]:
+        # FIXME add an override for JIT and revert 0. back to 0
```

Comment:
Would you file an issue for this and link the issue, too?

Reply:
yes will do